floss/language/utils.py

```diff
@@ -473,6 +473,43 @@ def get_struct_string_candidates(pe: pefile.PE) -> Iterable[StructString]:
             # dozens of seconds or more (suspect many minutes).
 
 
+def get_raw_xrefs_rdata_i386(pe: pefile.PE, buf: bytes) -> Iterable[VA]:
+    """
+    scan for raw xrefs in .rdata section.
+    raw xrefs are 32-bit absolute addresses to strings in .rdata section (i386).
```

Comment:
> raw xrefs are 32-bit absolute addresses to strings in .rdata section (i386).

This routine doesn't  validate that the destination is string-like data. Should it? If not, lets remove this part of the documentation.